examples/react-router/app/routes/json.tsx

```
@@ -0,0 +1,126 @@
+						return JSON.stringify(value);
+					}
+					return defaultSerialize(value);
+				},
+			}) ?? false,
+	);
+
+	return (
+		<div>
+			<Form {...form.props} method="post">
+				<div>
+					<label>Title</label>
```

Biome (2.4.6)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)